src/Uno.Extensions.Core.Generators/Uno.Extensions.Core.Generators.csproj

```diff
@@ -15,12 +15,14 @@
 		<Content Include="buildTransitive\Uno.Extensions.Core.props" />
 	</ItemGroup>
 
+	<ItemGroup>
+		<AdditionalFiles Include="AnalyzerReleases.Shipped.md" />
+		<AdditionalFiles Include="AnalyzerReleases.Unshipped.md" />
+	</ItemGroup>
```

Comment:
Somehow these were previously injected automatically but it is no longer the case after removing `PackageReference`s. So injecting them manually.